src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/prisonerprofile/PrisonerProfileService.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.prisonerprofile
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.prisonerprofile.api.PrisonerPhysicalAttributesResponse
+
+@Service
+class PrisonerProfileService {
+  // TODO SDIT-1817 Implement this service
+  fun getPhysicalAttributes(prisonerNumber: String) = PrisonerPhysicalAttributesResponse(prisonerNumber, listOf())
+}
```

Comment:
I just wanted to get the API and response definitions in - will look at implementing the service next.